viadot/tasks/vid_club.py

```diff
@@ -33,27 +35,31 @@ def __init__(
         Task to downloading data from Vid Club APIs to Pandas DataFrame.
 
         Args:
-            source (str): The endpoint source to be accessed, has to be among these:
-                ['jobs', 'product', 'company', 'survey'].
+            source (Literal["jobs", "product", "company", "survey"], optional): The endpoint source to be accessed. Defaults to None.
             credentials (Dict[str, Any], optional): Stores the credentials information. Defaults to None.
+            credentials_secret (str, optional): The name of the secret in Azure Key Vault or Prefect or local_config file. Defaults to "VIDCLUB".
+            vault_name (str, optional): For credentials stored in Azure Key Vault. The name of the vault from which to obtain the secret. Defaults to None.
             from_date (str): Start date for the query, by default is the oldest date in the data, '2022-03-22'.
-            to_date (str): End date for the query, if empty, datetime.today() will be used.
-            if_empty (str, optional): What to do if query returns no data. Defaults to "warn".
-            retry_delay (timedelta, optional): The delay between task retries. Defaults to 10 seconds.
+            to_date (str, optional): End date for the query. By default, datetime.today() will be used.
```

Comment:
You provided information that `datetime.today()` is a default. In code default is `None`. Maybe assign `datetime.today()` as a default value.

Reply:
ok

Reply:
Replace in the rest of the places if needed.